writings-volume-1/updated/02-chapter-1.adoc

```diff
@@ -615,6 +615,7 @@ and satisfies it with the true, precious substance forevermore.`"
 
 [.signed-section-signature]
 Isaac Penington
+//and this one?
```

Comment:
Same thing here... this whole testimony should be embedded. It starts up a few pages with: “My heart from my childhood was pointed towards the Lord..."